playwright/e2e/platform-infra/akamai-redirects.spec.ts

```
@@ -0,0 +1,35 @@
+import { test, expect, APIRequestContext } from '@playwright/test';
```

ESLint

[error] Member 'expect' of the import declaration should be sorted alphabetically.

(sort-imports)